### PR TITLE
update whitelist

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

**Comment by yuwata** [comment]
@masami256 Updated. PTAL.

File: whitelist/recipes-whitelist.txt [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
-binutils-cross-arm
-core-image-minimal
-cross-localedef
-cwautomacros
-depmodwrapper-cross
-dwarfsrcfiles
-eudev
-gcc-cross-arm
-gcc-runtime
-gettext-minimal
```
Comment (hiraku-wfs) [inline whitelist/recipes-whitelist.txt:13]: why are these files removed?

Comment (yuwata) [reply whitelist/recipes-whitelist.txt:13]: This section now only contains conf or script only packages. These removed packages contain some binaries, or already debianized thus not necessarily listed.

Comment (hiraku-wfs) [reply whitelist/recipes-whitelist.txt:13]: I understand your intention. But could you remove only debianized packages? I'd like to keep other packages listed here.

**Comment by yuwata** [comment]
@hiraku-wfs Updated. PTAL.

**Comment by hiraku-wfs** [comment]
Thank you!
